internal/envcheck/tasks.go

```diff
@@ -0,0 +1,73 @@
+package envcheck
+
+import (
+	"errors"
+	"os"
+	"strings"
+
+	"golang.org/x/sys/execabs"
+)
+
+type task struct {
+	name, hint string
+	test       func() (string, error)
+}
+
+var tasks = []*task{
+	{"Go compiler", "Checking Go is installed and up to date", func() (string, error) {
+		path, err := execabs.LookPath("go")
+		if err != nil {
+			return "", err
+		}
+
+		cmd := execabs.Command(path, "version")
+		ret, err := cmd.Output()
+		if err != nil {
+			return "", err
+		}
+		return strings.TrimSpace(string(ret)), nil
+	}},
+	{"C compiler", "Checking C is installed for system code", func() (string, error) {
```

Comment:
```suggestion
	{"C compiler", "Checking C-compiler is installed for system code", func() (string, error) {
```

Reply:
There might be an even better way of wording it. I just feel like installing "C" is misleading.

Reply:
Thanks, improved it now I think